alkira/resource_alkira_service_fortinet.go

```diff
@@ -68,6 +86,7 @@ func resourceAlkiraServiceFortinet() *schema.Resource {
 								"Required only when licenseType is BRING_YOUR_OWN.",
```

Comment:
This comment needs to be reformed.

Required only when `license_type` is `BRING_YOUR_OWN`.